pandas/tests/scalar/timedelta/test_arithmetic.py

```diff
@@ -616,3 +680,14 @@ def test_rdivmod_invalid(self):
 
         with pytest.raises(TypeError):
             divmod(np.array([22, 24]), td)
+
+    @pytest.mark.parametrize('op', [operator.mul, ops.rmul, operator.truediv, ops.rdiv])
+    @pytest.mark.parametrize('arr', [
+                                      [Timestamp('20130101 9:01'),
```

Comment:
shouldn't these be np.arrays, not lists?

Reply:
good catch, done